composer.json

```diff
@@ -18,7 +18,7 @@
         "php": "7.*",
         "ext-mbstring": "*",
         "ext-tokenizer": "*",
-        "nikic/php-parser": "^2.1|^3.0"
+        "nikic/php-parser": "^4.0"
```

Comment:
Why don't you just add `^4.0` to `^2.1|^3.0` instead of replacing it, if it's compatible with all three major versions?

Reply:
v2 makes no sense to have it as we minimum require PHP 7, but we can stay with v3 :smile: